src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/client/dto/AllocationDemandDetails.kt

```diff
@@ -50,4 +50,20 @@ data class Requirement(
   val subCategory: String,
   val length: String,
   val id: BigInteger,
+  val manager: Manager,
+  val isUnpaidWork: Boolean,
+)
+
+data class Manager(
+  val code: String,
+  val teamCode: String,
+  val grade: String,
+  val name: ManagerName,
+  val allocated: Boolean,
+)
+
+data class ManagerName(
```

Comment:
We could probably use the existing Name class which is already used in the parameters for the AllocationDemandDetails constructor